pyproject.toml

```diff
@@ -285,7 +289,7 @@ max-args = 8
 classmethod-decorators = ["pydantic.validator", "pydantic.root_validator"]
 
 [tool.ruff.lint.per-file-ignores]
-"tests/**.py" = [  # Some things we want for the moin project are unnecessary in tests.
+"tests/**.py" = [  # Some things we want for the main project are unnecessary in tests.
```

Comment:
This particular typo, switching a and o, is probably the biggest downside for me of dvorak